### DISCUSSION TITLE
Ebyte E220 usage?

### DISCUSSION THREAD

**Discussion by desultory** [tapparelj/gr-lora_sdr#106] [Q&A]
How am I supposed to get this to decode these messages? I can see the transmission on the waterfall but can't get it to decode it. I don't think the spreading factor on these devices can be adjusted and i've tried a handful. The data rate should be correct (2.4k)
![image](https://github.com/user-attachments/assets/a10f5cda-3650-4211-bd68-299d5971f663)

**Comment by jasemanoah** [comment]
![Screenshot from 2025-02-20 12-49-13](https://github.com/user-attachments/assets/3e4644b7-2b29-477d-b7d6-baca321bf315)
Hey, Can you help me out with this, I'm using a bladerf to receive rf using lora receiver but i get an error saying 'could not find port ascii in" also sometimes as "no message input".. 
![Screenshot from 2025-02-20 12-49-13](https://github.com/user-attachments/assets/b4ee6632-afef-4ccf-b1d0-7a30fb9e1081)
Would be grateful for your help... you can email me at <jasonmanoah2409@gmail.com>
